test/benchmark/avro-upsert/mzcompose.yml

```diff
@@ -126,6 +126,34 @@ mzworkflows:
     - step: run
       service: create-views
 
+  benchmark-medium:
+    steps:
+    - step: start-services
+      services: [dashboard]
+    - step: start-services
+      services: [kafka, schema-registry, materialized]
+    - step: wait-for-tcp
+      host: kafka
+      port: 9092
+    - step: wait-for-tcp
+      host: schema-registry
+      port: 8081
+    - step: run
+      service: avro-upsert-generator
+      command: >-
+        --parallelism=40
```

Comment:
can we tune the parallelism down - i only have 8 cores

Reply:
We can tune it down to 8 if you like -- the main thing is to make sure we keep the number of messages evenly divisible by the number of threads, otherwise the generator won't write enough data and the benchmark will never pass.

Reply:
sure - it seems like we could maybe teach the generator script to complain if the number of requested messages isnt divisible by the parallelization factor?

Reply:
You read my mind! I'm just writing that up now :)